lib/include/tick/solver/sto_solver.h

```diff
@@ -67,10 +68,20 @@ class TStoSolver {
   void init_permutation();
 
  public:
-  explicit TStoSolver(int seed = -1);
-
-  TStoSolver(ulong epoch_size = 0, T tol = 0.,
-             RandType rand_type = RandType::unif, int seed = -1);
+  // inline TStoSolver(int seed = - 1) : seed(seed) {
+  //     set_seed(seed);
+  // permutation_ready = false;
+  // }
```

Comment:
We can remove it but we shouldn't leave it commented imo

Reply:
done